odxtools/parentref.py

```diff
@@ -10,7 +10,7 @@
 from .utils import dataclass_fields_asdict
 
 if TYPE_CHECKING:
-    from .diaglayer import DiagLayer
```

Comment:
keep a placeholder diaglayer.py file that just imports diaglayers.diaglayer content, in order to reduce the ammout of unrelated refactoring in this PR

Reply:
I think that this would only make things more confusing because most of the import-related code churn stems from the fact that additional classes need to be imported and IMO the canonical locations should be used for these anyway...

Reply:
AFAICS this would not work very well anyway, because the attributes of the specialized diag layers have changed. I think we should just make the next release 8.0. (I suppose the number of users which would be saved from trouble due to this is very small.)

Reply:
okay: I just released v7.4.0 to have a release which contains the fixes which have gone into `main` over the last few weeks but without the potentially disruptive diaglayer changes (which will only go into v8.0)